source/posts/7070-07-07-start-and-stop-your-work-day-with-a-list.html.md

```diff
@@ -0,0 +1,104 @@
+attention. I've also taken care of myself by doing things that set me up for an
+effective day.
+
+## I Don't Know What Some of These Items Mean
+
+Yeah, some of these are probably a bit opaque but it doesn't really matter - the
+key here is that I have a list where I can put things that I want to do every
+day, what those things are can change over time!
+
+What I like to recommend to people is that you have this type of habit and list,
+not that you match my items - that's not what's important.
+
+## Ok What About the Stop List
+
+The end of my day is almost always right around 5:00 and I have a notification
+that fires at 5 minutes before that time. That is my trigger to wrap up what I'm
+doing and switch back over to Todoist and find the recurring item called "run
+stop list". Here's how that one looks:
+
+* say goodbye in Slack
+* quit Slack and Mail
+* verify times in Forty
+* review Todoist for completed items
+* prepare start list
+* check time of first meeting tomorrow
+* close dashboards
+* mute computer
+* plug in mouse
+
+## And You Really do This Every Day?
+
+Ok you got me, there are days I don't follow this habit. Some days I dive into
+Slack before doing the routine. Other days maybe I'm running late and head out
+before even opening my stop list. That's ok, habits are not all or none - I get
+tons of value out of striving for this habit regardless of how perfect I am
+about it. What I've noticed is that my best days probably began with my start
+list and I do a better job of transitioning away from work when I run my stop
+list.
+
+On a typical day the start list might take me 20 minutes to run through
+(especially on Mondays) and the stop list is about 5 minutes.
+
+## Is There an MVP of This Habit?
+
+If you want to start building a habit like the one I've just described, here's
+what I'd recommend you start with:
+
+Start List
+
+* review calendar: look at the meetings on your calendar - are you prepared for
+  them? Can you cancel any of them?
+* open chat app: wait to open chat until you're ready.
+* process email: go through email, reply to and archive messages that take 1
+  minute or less to deal with, anything else going on your todo list.
+
+Stop List
+
+* quit chat app and email: begin winding down by quitting the most distracting
+  things in your life. Don't let them be the first thing you see when you sit
+  down tomorrow morning.
+* check time of first meeting tomorrow: avoid being surprised by first-thing
+  meetings by checking for the time of your first meeting the next day.
+
+## What Else Should I Add?
+
+My lists have evolved over time to include hints, prompts, and nudges that I
+find useful - you should be on the look out for these too! An item on these
+lists can be anything but incorporating this basic flow into your life is what
+sets you up to build upon it and that's where the real power comes from.
+
+[Todoist]: https://todoist.com/r/jonathan_allured_mzkvzo
```

Comment:
is this intended to take you to a user profile or something? It takes me to the todoist home page. 

I suspect this is accidental but just pointing out that it's got some user-id looking thing that might be best not exposed.

Reply:
Yep, that was a mistake, thanks!! 🤗